Dockerfile

```diff
@@ -9,6 +9,4 @@ COPY . .
 
 RUN make install
 
-EXPOSE 3000
-
-CMD ["pnpm", "run", "dev"]
+EXPOSE 3001
```

Comment:
You need to remove this line

Reply:
if i remove this line container is running over and over again